docs/README.md

```diff
@@ -0,0 +1,7 @@
+# AeroGear Android SDK Documentation
+
+This directory contains the preliminary documentation for the AeroGear Android SDK and will extended during development.
```

Comment:
`and will extended during development.` This may be not needed as we will need to remove it after.